Client/Frontend/Browser/Playlist/Controllers/PlaylistNewFolderView.swift

```diff
@@ -37,6 +37,13 @@ class PlaylistFolderImageLoader: ObservableObject {
     loadImage(url: favIconUrl, isFavIcon: true)
   }
 
+  func load(domainUrl: URL) {
+    let fetcher = FavIconImageRenderer()
+    fetcher.loadIcon(siteURL: domainUrl) { [weak self] image in
+      self?.image = image
```

Comment:
`Fetcher` will go out of scope and cancel the `loadIcon` call. Should retain until the call completes or retain `fetcher`?